### ISSUE TITLE
kgtk text_embedding error

### ISSUE THREAD

**Issue by dgarijo** [usc-isi-i2/kgtk#164]
**Describe the bug**
In the Example 1 (embeddings) I got an error when executing:

```
kgtk import_conceptnet --english_only conceptnet-assertions-5.7.0.csv / \
            filter -p " ; /r/Causes,/r/UsedFor,/r/Synonym,/r/DefinedAs,/r/IsA ; " / sort -c 1,2,3 \
            | head -30000 |
            kgtk text_embedding --debug --embedding-projector-metadata-path none \
                    --embedding-projector-metadata-path none \
                    --label-properties "/r/Synonym" \
                    --isa-properties "/r/IsA" \
                    --description-properties "/r/DefinedAs" \
                    --property-value "/r/Causes" "/r/UsedFor" \
                    --has-properties "" \
                    -f kgtk_format \
                    --output-format kgtk_format \
                    --use-cache \
                    --model bert-large-nli-cls-token \
                    > emb.txt 
```
Error: 
```
100%|██████████| 1.24G/1.24G [1:03:58<00:00, 324kB/s]   
Traceback (most recent call last):
  File "/opt/conda/lib/python3.7/site-packages/kgtk-0.4.0-py3.7.egg/kgtk/exceptions.py", line 42, in __call__
    return_code = func(*args, **kwargs) or 0
  File "/opt/conda/lib/python3.7/site-packages/kgtk-0.4.0-py3.7.egg/kgtk/cli/text_embedding.py", line 332, in run
    main(**kwargs)
  File "/opt/conda/lib/python3.7/site-packages/kgtk-0.4.0-py3.7.egg/kgtk/cli/text_embedding.py", line 208, in main
    property_labels_dict=property_labels_dict)
  File "/opt/conda/lib/python3.7/site-packages/kgtk-0.4.0-py3.7.egg/kgtk/gt/embedding_utils.py", line 405, in read_input
    raise KGTKException("Missing column: {}".format(missing_column))
kgtk.exceptions.KGTKException: Missing column: {'label'}
Missing column: {'label'}
```
It took a while, so I would say the error happened when the results were dumped into disk.

**To Reproduce**
I ran the version in the master branch and executed example 1 through a Docker container.

**Expected behavior**
The error should not appear.

**Comment by dgarijo** [comment]
Note: import conceptnet works fine. I am testing now which of the commands is leading to the error.

**Comment by dgarijo** [comment]
More information about this issue:
I have verified that the error was on `kgtk text_embedding`. Apparently the previous commands add the column `relation` instead of `label`. By doing `sed -i '1!b;s/relation/label/' heads.kgtk` I created a file replacing `relation` with `label`. The embeddings are still calculating on a server, but it looks like the error is gone (before it didn't get this far in the console log).

So the error to fix is either make sure that there is no `relation` in the generated file, or change the `kgtk text_embedding` to be flexible and accept KGTK files with `relation` instead of `label`
